AGGREGATE.md

```diff
@@ -209,8 +213,26 @@ The browser will encrypt payloads just before the report is sent by requesting p
 
 **Note:** The browser may need some mechanism to ensure that the same set of keys are delivered to different users.
 
-**TODO**: more formally specify `payload`.
+The contents of the base64-encoded encrypted payload will be a [CBOR](https://cbor.io) map.
+
+Payloads generated with the `two-party` processing type will have the following structure:
+
+```
+// CBOR
+{
+  "version": "<api version>",
+  "operation": "hierarchical-histogram",  // Allows for the service to support other operations in the future
+  "privacy_budget_key": "<field for server to do privacy budgeting>",
+  "scheduled_report_time": <timestamp in msec>,
+  "reporting_origin": "https://reporter.example",
+  "dpf_key": <binary serialization of the DPF key>,
+}
+```
+Note that the `dpf_key` value would differ in the two payloads.
 
+Payloads generated with the `single-server` processing type will directly encode the bucket and value of the histogram contribution instead of a DPF key.
+The `dpf_key` field will be replaced with `"data": { "bucket": <bucket>, "value": <value> }`.
```

Comment:
I wonder if it makes sense to encode bucket as a string here, it's possible we may want the schema to be robust to output domain size.

Reply:
The payload is currently CBOR, which supports integers in [-2^64, 2^64). Do we expect bucket sizes might go beyond 64-bit unsigned/65-bit signed? I'm leaning towards leaving it as is for efficiency and because supporting >64-bit integers will already require some non-trivial changes to the parsing/processing code (so changing to a string then won't be too bad). Wdyt?

Reply:
Ack. Given the explainer uses 32 bits as the example, this seems fine. I can file a separate issue to track this as it probably affects other parts as well.